### ISSUE TITLE
Tests don't run properly

### ISSUE THREAD

**Issue by scottgonzalez** [tessel/servo-pca9685#15]
It seems that either the tests don't run properly or there's more needed than just running `npm test`.

From a fresh clone of master, after installing dependencies, this is the output I get:

```
servo-pca9685 (master) $ npm test

> servo-pca9685@0.1.3 test /Users/scottgonzalez/Projects/servo-pca9685
> tinytap -e 'tessel run {} ${SERVO_PORT}' test/*.js

1..2
not ok 1 - test/apiTest.js
not ok 2 - test exited with code 1
npm ERR! Test failed.  See above for more details.
```

**Comment by johnnyman727** [comment]
@scottgonzalez can you confirm that this is on a T2?

**Comment by scottgonzalez** [comment]
Yes, though it's entirely possible that it just wasn't connecting given all the USB issues I'm having with my device.

**Comment by scottgonzalez** [comment]
It's also worth noting that there are no instructions on how to run the tests, so I wasn't sure if the Tessel even needed to be plugged in. I first tried without it plugged in and when I got the above output, I plugged it in, waited for it to fully boot, then tried again, but got the same results.
